js_modules/dagit/packages/core/src/assets/AssetNodeDefinition.tsx

```diff
@@ -149,8 +149,8 @@ const JobGraphLink: React.FC<{
   return (
     <Link
       to={instanceAssetsExplorerPathToURL({
-        opNames: [token],
-        opsQuery: direction === 'upstream' ? `*${token}` : `${token}*`,
+        opNames: [],
```

Comment:
Tbh, I wasn't sure if we should leave this as selected when viewing downstream assets. The complaint in #6839 is about upstream, but I do think if you've just arrived at this graph showing a bunch of assets, the default should be to rematerialize everything you're seeing, regardless of up/down direction.

Reply:
Yep I think this makes sense, we should definitely default to the "whole graph" both up / downstream